src/main/telemetry/crsf.c

```diff
@@ -406,6 +406,8 @@ void crsfFrameFlightMode(sbuf_t *dst)
         flightMode = "MANU";
     } else if (FLIGHT_MODE(ANGLE_MODE)) {
         flightMode = "STAB";
+    } else if (FLIGHT_MODE(ALTHOLD_MODE)) {
```

Comment:
See above.

Reply:
Here I have hesitated to change `ALTHOLD_MODE` to `ALT_HOLD_MODE` because there are so many instances, but yes, you're right.  Time to do that also.

Reply:
Think @ledvinap means line 410 here

Reply:
Ah OK I've done both... just. fixed this one.